agent/util-scripts/pbench-tool-meister-start

```diff
@@ -283,6 +284,9 @@ def main(argv):
         logger.exception("failed to load tool group data")
         return 1
 
+    # See if anybody told us to use certain options with SSH commands.
+    ssh_opts = os.environ.get("ssh_opts", "")
```

Comment:
Right now this CLI orchestrator does not really use the `PbenchAgentConfig` class directly, and because we have published the environment variable, somebody can override the settings anyway.  If they do override, the bench-scripts would behave one way, and the Tool Meister startup would behave another.

Reply:
OK